map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -174,27 +175,13 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
             setLatLngBoundsForCameraTarget(museumBounds)
         }
 
-        // TODO: use custom tiling here instead.
-        baseGroundOverlay = map.addGroundOverlay(
-                GroundOverlayOptions()
-                        .positionFromBounds(
-                                LatLngBounds(
-                                        LatLng(41.874620, -87.629243),
-                                        LatLng(41.884753, -87.615841)
-                                )
-                        ).image(deriveBaseOverlayDescriptor(requireActivity()))
-                        .zIndex(0.1f)
-        )
-        buildingGroundOverlay = map.addGroundOverlay(
-                GroundOverlayOptions()
-                        .positionFromBounds(
-                                LatLngBounds(
-                                        LatLng(41.878467, -87.624127),
-                                        LatLng(41.880730, -87.621027)
-                                )
-                        ).zIndex(.11f)
-                        .image(BitmapDescriptorFactory.fromAsset("AIC_Floor1.png"))
-        )
+        tileOverlay = map.addTileOverlay(TileOverlayOptions()
+                .zIndex(0.2f)
+                .tileProvider(GlideMapTileProvider(requireContext(),1)))
+
+//        map.addTileOverlay(TileOverlayOptions().zIndex(0.3f)
```

Comment:
This is still needed for future work, as @raigex tells me.

Reply:
correct